### PR TITLE
Day 2 ✅ 

### PR DESCRIPTION
**Pull request by camargodev**

### CODE REVIEW

File: day-2/part-2/part-2.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+    for i in range(int(len(results)/2)):
+        result = int(results[2*i])
+        color = results[2*i + 1]
+        results_by_color[color].append(result)
+
+    return results_by_color
+
+def calculate_min_product(results_by_color):
+    return math.prod([max(results) for results in results_by_color.values()])
```
Comment (jdalbosco) [inline day-2/part-2/part-2.py:31]: Nice, looks clean

File: day-2/part-1/part-1.py [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    return int(game_id), results
+
+def parse_game_info(line):
+    results_by_color = {"red": [], "green": [], "blue": []}
+    game_id, results = read_raw_data_info(line)
+
+    for i in range(int(len(results)/2)):
+        result = int(results[2*i])
+        color = results[2*i + 1]
+        results_by_color[color].append(result)
```
Comment (jdalbosco) [inline day-2/part-1/part-1.py]: As we discussed - this makes sense if you know how the input looks like but it's not very readable IMO

File: day-2/part-2/part-2.py (hunk 1 above)
Comment (camargodev) [reply day-2/part-2/part-2.py:31]: Thx!

File: day-2/part-1/part-1.py (hunk 2 above)
Comment (camargodev) [reply day-2/part-1/part-1.py]: Improved at [4778935](https://github.com/camargodev/advent-of-code-2023/pull/2/commits/47789354e419f17bd1754d287abde3bce30e1ee7), please take a look.

**Review by jdalbosco — APPROVED** [review]

**Pull request merged**